### ISSUE TITLE
Error in master

### ISSUE THREAD

**Issue by inc0der** [rpgtkoolmv/corescript#170]
A freshly cloned repository and then a new build of all files `npm run build` causes the game to not even start do to the @triacontane patch.

![image](https://user-images.githubusercontent.com/14813394/42704204-5dbaa1b2-86a5-11e8-8ee9-bb67bfa16e83.png)

Not sure why this is happening, I"m using a new project using MV 1.6.1 to build a new project in the `/game/` directory of the repository. It must only be happening for me so it must be something on my side otherwise you guys wouldn't be merging PR's to master.

**Comment by inc0der** [comment]
Okay it's because  it's looking for the tilemap but the variable is refrencing `tile`

```js
var TileRenderer                  = PIXI.WebGLRenderer.__plugins.tile;
```
and it should be 
```js
var TileRenderer                  = PIXI.WebGLRenderer.__plugins.tilemap;
```

This also happens within `ShaderTilemap.js` in the method `_hackRenderer`. Changing it to tilemap seems to have fixed it so for now I'm going to assume this is because I'm using 1.6.1 and continue checking things out.

**Comment by krmbn0576** [comment]
Yes, as you pointed out, `1.6.1` and `master` branch are incompatible now. 😢 
I want to solve this situation in a while.

**Comment by inc0der** [comment]
This is all I had to do to get the game running in 1.6.1.  If you want I can make a PR for it? https://github.com/ltngames/corescript/commit/07af16a1a6edaabba2a8c376b6fcbc8ca4a23a0f
I'm not sure what the patch was supposed to do but everything seems to run fine in my project with this fix, no issues with the tilemap, and everything runs smooth.

**Comment by krmbn0576** [comment]
Fixed by #193
